src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/Flyout/Flyout_ShowAt_Window_Content.xaml.cs

```diff
@@ -13,9 +15,20 @@ namespace UITests.Windows_UI_Xaml_Controls.FlyoutTests
 	[Sample("Flyouts")]
 	public sealed partial class Flyout_ShowAt_Window_Content : Page
 	{
+		private XamlRoot _xamlRoot;
+
 		public Flyout_ShowAt_Window_Content()
 		{
 			this.InitializeComponent();
+
+#if HAS_UNO
+			Unloaded += (s, e) => _xamlRoot = XamlRoot;
+			Unloaded += (s, e) =>
```

Comment:
Was the intention to subscribe to the same event twice with different actions?

Reply:
Indeed, thanks!